Tools/build-translation.js

```diff
@@ -56,13 +56,26 @@ function buildLocale(inputFile, outputFile) {
 	saveToFile(outputFile, translation);
 }
 
-async function removePoHeaderDate(filePath) {
-	// Note: on macOS this will fail because it needs to be 'sed -i ""'
-	// Solution would be to install gsed, detect it here, and use it in place of sed in macOS
-	// https://stackoverflow.com/questions/30003570/how-to-use-gnu-sed-on-mac-os-x#34815955
+function returnPath(file) {
+	const potentialPaths = [
+		'/usr/local/opt/gettext/bin/',
+		'/opt/local/bin/',
+		'/usr/local/bin/',
+	];
+
+	for (const path of potentialPaths) {
+		let pathFile = path + file;
+		if (fs.existsSync(pathFile)) {
+			return pathFile;
+		}
+	}
+	console.log(file + ' could not be found. Please install via brew or MacPorts.');
+	process.exit(1);
```

Comment:
Please throw an exception, which should be caught in main().

Reply:
Ok, will do. I haven't done it at first, because I don't think a stack trace is needed for a script.

Reply:
Yes, possibly. It's just the pattern used in all scripts so far. I make a main() entry point where all issues can be caught.

Reply:
For sake of consistency this makes perfect sense then.

I love stack traces, it's just that they can be rather intimidating for users, but since most people won't run these commands anyway, all's good.